invenio_vocabularies/contrib/awards/mappings/v7/awards/award-v1.0.0.json

```diff
@@ -0,0 +1,79 @@
+{
+  "mappings": {
+    "dynamic": "strict",
+    "properties": {
+      "$schema": {
+        "type": "keyword",
+        "index": "false"
+      },
+      "created": {
+        "type": "date"
+      },
+      "id": {
+        "type": "keyword"
+      },
+      "updated": {
+        "type": "date"
+      },
+      "uuid": {
+        "type": "keyword"
+      },
+      "version_id": {
+        "type": "integer"
+      },
+      "identifiers" : {
+        "properties": {
+          "identifier" : {
+            "type" : "keyword"
+          },
+          "scheme" : {
+            "type" : "keyword"
+          }
+        }
+      },
+      "pid": {
+        "type": "object",
+        "properties": {
+          "pk": {
+            "type": "integer"
+          },
+          "pid_type": {
+            "type": "keyword"
+          },
+          "obj_type": {
+            "type": "keyword"
+          },
+          "status": {
+            "type": "keyword"
+          }
+        }
+      },
+      "title": {
+        "type": "object",
+        "dynamic": true,
+        "properties": {
+          "en": {
+            "type": "search_as_you_type"
+          }
+        }
+      },
+      "number": {
+        "type": "keyword"
+      },
+      "funder": {
+        "type": "object",
+        "properties": {
+          "@v": {
+            "type": "keyword"
+          },
+          "id": {
+            "type": "keyword"
+          },
+          "name": {
+            "type": "keyword"
```

Comment:
keyword or text?

Reply:
I think text would be the correct one. Nonetheless, `names` vocabulary also defines `affiliations` names as `keyword`. We might want to change that as well?

Reply:
Yes, but not in the same PR. Could you open a quick issue (no need for a big description) and PR, and add the issue to the board?

Reply:
Created it here https://github.com/inveniosoftware/invenio-vocabularies/issues/166